src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -2154,9 +2154,26 @@ private ISwitchExpressionOperation CreateBoundSwitchExpressionOperation(BoundSwi
         {
             IOperation value = Create(boundSwitchExpression.Expression);
             ImmutableArray<ISwitchExpressionArmOperation> arms = CreateFromArray<BoundSwitchExpressionArm, ISwitchExpressionArmOperation>(boundSwitchExpression.SwitchArms);
+
+            bool isExhaustive;
+            if (boundSwitchExpression.ReportedNotExhaustive)
```

Comment:
We don't want to base our logic off this property at all. The only thing we should be checking is whether the default label was generated.